apps/src/templates/instructions/MarkdownInstructions.jsx

```diff
@@ -53,6 +55,22 @@ const MarkdownInstructions = React.createClass({
       });
     }
 
+    if (this.props.isBlockly) {
+      // Convert any inline XML into blockly blocks. Note that we want to
+      // make sure we don't initialize any blockspace before the main
+      // block space has been created, lest we violate some assumptions
+      // blockly has.
```

Comment:
_so many_

I spent some time trying to track them all down, but couldn't. We could consider this a tech debt item, but easily a substantial one.